binancial/strategies/mean_reversion_crossover.py

```
@@ -1,4 +1,4 @@
-def mean_strategy(ticker, accountant, price_usdt, timestamp):
+def mean_reversion_crossover(ticker, accountant, price_usdt, timestamp):
```

Ruff (0.8.2)

Missing return type annotation for public function `mean_reversion_crossover`

(ANN201)